### DISCUSSION TITLE
Akva - Att skapa förutsättningar för nya digitala tjänster, till exempel medborgarforskning.

### DISCUSSION THREAD

**Discussion by salgo60** [salgo60/Svenskabadplatser#132] [Show and tell]
Hur kan Wikidata öppna upp detta jmf hur jag gjort det möjligt att bilder uppladdade på Wikicommons kan enkelt kopplas till vilken runsten den avbildar hos Riksantikvarieämbetet [SPARQL bilder](https://wcqs-beta.wmflabs.org/embed.html#%23defaultView%3AImageGrid%0ASELECT%20DISTINCT%20%3Fsignum%20%3Ffile%20%3Fitem%20%3Fimage%20%3Fksam%0AWITH%20%0A%7B%20SELECT%20%3Fitem%20%3FitemLabel%20%3FitemDescription%20%3Fksam%20%3FEvighetsRunor%20%3Fsignum%20WHERE%0A%20%20%7B%20SERVICE%20%3Chttps%3A%2F%2Fquery.wikidata.org%2Fsparql%3E%20%0A%20%20%20%20%7B%3Fitem%20wdt%3AP1261%20%3Fsignum%3B%0A%20%20%20%20%20%20wdt%3AP1260%20%3Fksam.%0A%20%20%20%20%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22sv%2Cen%22.%20%3Fitem%20rdfs%3Alabel%20%3FitemLabel%20.%7D%0A%20%20%20%20%20%20FILTER%28CONTAINS%28%3Fksam%2C%22uu%2Fsrdb%2F%22%29%29%0A%20%20%20%20%20BIND%28URI%28CONCAT%28%22http%3A%2F%2Fkulturarvsdata.se%2F%22%2C%3Fksam%29%29%20AS%20%3FEvighetsRunor%29%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%20AS%20%25Wikidataitems%0A%0AWHERE%20%0A%7B%20%20INCLUDE%20%25Wikidataitems%20.%0A%20%20%3Ffile%20wdt%3AP180%20%3Fitem.%0A%20%20%3Ffile%20schema%3AcontentUrl%20%3Furl.%20%0A%20%20bind%28iri%28concat%28%22http%3A%2F%2Fcommons.wikimedia.org%2Fwiki%2FSpecial%3AFilePath%2F%22%2C%20wikibase%3AdecodeUri%28substr%28str%28%3Furl%29%2C53%29%29%29%29%20AS%20%3Fimage%29%0A%7D) / [SPARQL karta](https://wcqs-beta.wmflabs.org/embed.html#%23defaultView%3AMap%0ASELECT%20DISTINCT%20%3Fsignum%20%3Ffile%20%3Fitem%20%3Fimage%20%3Fksam%20%3Fcoord%0AWITH%20%0A%7B%20SELECT%20%3Fitem%20%3FitemLabel%20%3FitemDescription%20%3Fksam%20%3FEvighetsRunor%20%3Fsignum%20%3Fcoord%20WHERE%0A%20%20%7B%20SERVICE%20%3Chttps%3A%2F%2Fquery.wikidata.org%2Fsparql%3E%20%0A%20%20%20%20%7B%3Fitem%20wdt%3AP1261%20%3Fsignum%3B%0A%20%20%20%20%20%20wdt%3AP1260%20%3Fksam.%0A%20%20%20%20%20%20%3Fitem%20wdt%3AP625%20%3Fcoord.%0A%20%20%20%20%20%20SERVICE%20wikibase%3Alabel%20%7B%20bd%3AserviceParam%20wikibase%3Alanguage%20%22sv%2Cen%22.%20%3Fitem%20rdfs%3Alabel%20%3FitemLabel%20.%7D%0A%20%20%20%20%20%20FILTER%28CONTAINS%28%3Fksam%2C%22uu%2Fsrdb%2F%22%29%29%0A%20%20%20%20%20BIND%28URI%28CONCAT%28%22http%3A%2F%2Fkulturarvsdata.se%2F%22%2C%3Fksam%29%29%20AS%20%3FEvighetsRunor%29%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%20AS%20%25Wikidataitems%0A%0AWHERE%20%0A%7B%20%20INCLUDE%20%25Wikidataitems%20.%0A%20%20%3Ffile%20wdt%3AP180%20%3Fitem.%0A%20%20%3Ffile%20schema%3AcontentUrl%20%3Furl.%20%0A%20%20bind%28iri%28concat%28%22http%3A%2F%2Fcommons.wikimedia.org%2Fwiki%2FSpecial%3AFilePath%2F%22%2C%20wikibase%3AdecodeUri%28substr%28str%28%3Furl%29%2C53%29%29%29%29%20AS%20%3Fimage%29%0A%7D) eller vilka böcker hos Litteraturbanken som beskriver denna runsten och runstenen kopplas ihop med sida i boken [SPARQL](https://w.wiki/3Fz9) - [blog om detta](https://meta.wikimedia.org/wiki/Structured_data_for_GLAM-Wiki/Roundtripping/KMB) 

**AKVA** 
_Målet med programmet Nästa generation vatteninformation är att skapa förutsättningarna för, samt realisera delar av helhetsbilden av vattenmiljön i svensk havs- och vattenförvaltning._
![image](https://user-images.githubusercontent.com/14206509/120153284-d1225f00-c1ee-11eb-94ec-3034101de197.png)

* [video](https://www.youtube.com/watch?v=yDkVQ_PdA8w&t=1s)

* [Nästa Generation Vatteninformation](https://www.havochvatten.se/planering-forvaltning-och-samverkan/program-projekt-och-andra-uppdrag/nasta-generation-vatteninformation.html) - 

![image](https://user-images.githubusercontent.com/14206509/120153664-47bf5c80-c1ef-11eb-843d-f93e4d9a2dd7.png)

**Comment by soleildeminuit** [comment]
Intressant! Jag kände inte till detta viktiga initiativ. Av vad jag kan förstå av filmen har dock en av de viktigaste förutsättningarna glömts/inte tagits med - öppna data. Det räcker inte med en plattform med tjänster, det riskerar inlåsningseffekter. Vi saknar idag viktiga datamängder som öppna data, t.ex. botteninformation (batymetri). Hur som helst, @TGBMonsen - strategin visar att det kommer löna sig att harmonisera badvatten-specar med omvärlden...

Förresten kanske detta kan förklara varför många badplatser hamnat tokigt (koordinater som visas i en film ska väl åtminstone vara korrekta och autentiska) ;)
![image](https://user-images.githubusercontent.com/10654578/120155453-23647f80-c1f1-11eb-9b1c-82c6d2642eeb.png)

**Reply by salgo60** [reply]
:satisfied:
Vi får se jag tror på att kyssa många grodor och kanske dyker det upp en prins.... det jag tycker mig se med tanken att rapportera in badvatten från kommunerna så underskattas 9 ggr av 10 komplexiteten i detta med att leverera data som data hos dom flesta medans företag som Google, Amazon etc. rusar fram i överljudsfart så sitter vi och puttrar med svenska csv filer utan språkkod. Tror att Instagram har nästan lika bra badvattens platser som Hav - [762 kopplingar](https://w.wiki/3QpU) ... :unamused:


![image](https://user-images.githubusercontent.com/14206509/120157361-3d06c680-c1f3-11eb-9b07-af395d9ebf18.png)


Skriver lite om hur Wikivärlden kan vara en brygga mellan appar som iNaturalist och att Hav vill ha in observationer se snart https://github.com/salgo60/Svenskabadplatser/discussions/134

**Comment by ZeroPollution** [comment]
OK, koordinaterna är en miss... jag köper att jag borde kontrollerat dem, de skulle dock enbart ses som en symbol som illustrerar positioner.

Däremot så handlar Nästa Generation Vatteninformation (programmet som även tar fram produkten AKVA) till stor del om just öppna data. Vi har en historik att förhålla oss till, men vi är på väg. I våras lanserades vår metadatakatalog "Havskatten" internt och förväntas lanseras externt under hösten, nu pågår ett intensivt arbeta inom myndigheten att fylla på i katalogen. Data om badvatten är en del, men det finns så mycket mer information än så som vi vill tillgängliggöra. Det kommer att ta flera år innan vi är där vi vill vara.

Hör gärna av er om det finns frågetecken eller intresse kring detta.

**Reply by salgo60** [reply]
@ZeroPollution Enormt spännande vi har på Wikidata kopplat ihop oss med > [5000 externa källor](https://w.wiki/DYQ) men som sagt Wikidata är en öppen plattform med alla dess möjligheter och brister

Jag har enormt mycket intresse av detta.... har gjort [1 million redigeringar på Wikidata ](https://wikidata.wikiscan.org/user/Salgo60)och [pratade i förra månaden på LD4](https://youtu.be/GeDXzInR_mA?t=993) där jag bland annat pekade på min bakgrund som scrum master på SEB för deras internationella transaktioner dvs. metadata med 0 tolerans

* har även bara som mindre test skapar lite dataset för Svenska badplatser, [Svenska utegym](https://www.wikidata.org/wiki/Wikidata:WikiProject_Outdoor_Gyms) och det vi skulle vilja se av externa aktörer har jag sammanfattat i denna ostrukturertade lista "[One way to design a system to be a good external identifier in Wikidata](https://www.wikidata.org/wiki/User:Salgo60/ExternalIdentifiers)"

Jag nås på 
* salgo60@msn.com
* 0735152802
* [Linked in](https://www.linkedin.com/in/magnus-s%C3%A4lg%C3%B6-148890)

Lite förvirrade tankar se Issues [Havs och vattenmyndigheten](https://github.com/salgo60/Svenskabadplatser/issues?q=is%3Aissue+label%3A%22Havs-+och+vattenmyndighetens%22+)
